### DISCUSSION TITLE
autorestic as a Docker container?

### DISCUSSION THREAD

**Discussion by haneef95** [cupcakearmy/autorestic#128] [Q&A] [answered]
Hi,

This looks promising for what I need. Is this available as a docker container?

Thanks,

**Comment by cupcakearmy** [comment, marked as answer]
Not yes, but it should not be very complicated to do now that we have github actions. Feel free to open a ticket :)

**Reply by haneef95** [reply]
Thanks.

Done: https://github.com/cupcakearmy/autorestic/issues/129
